tgui/packages/tgui/interfaces/common/ElectricalPanel.tsx

```diff
@@ -0,0 +1,184 @@
+import { classes } from 'common/react';
+import { useBackend } from '../../backend';
+import { Box, Button, Icon, Flex, NoticeBox } from '../../components';
+import { BoxProps } from '../../components/Box';
+import { Table, TableCell, TableRow } from '../../components/Table';
+
+interface ElectricalData {
+  electrical: MachineElectrical;
+}
+
+interface MachineElectrical {
+  electrified: number;
+  panel_open: number;
+  wires: WireSpec[];
+  powered: number;
+}
+
+interface WireSpec {
+  desc: string;
+  cut: number;
+}
+
+const ElectricalPanelClosed = (props: BoxProps, context) => {
+  return (
+    <NoticeBox
+      className={classes([
+        'PanelClosed',
+        'ElectricalSafetySign',
+        props.className,
+      ])}>
+      <Flex
+        direction="row"
+        justify="space-between"
+        fill
+        className="ElectricalSafetySign">
+        <Flex.Item grow>
+          <Flex
+            justify="space-between"
+            direction="column"
+            fill
+            className={classes(['ElectricalSafetySign'])}>
+            <Flex.Item>
+              <Icon name="circle-xmark" />
+            </Flex.Item>
+            <Flex.Item>
+              <Icon name="circle-xmark" />
+            </Flex.Item>
+          </Flex>
+        </Flex.Item>
+        <Flex.Item fill>
+          <Flex
+            justify="space-around"
+            align="center"
+            inline
+            fill
+            wrap
+            className="WarningIcon"
+            direction="column">
+            <Flex.Item>
+              <Icon name="bolt" size={2} />
+            </Flex.Item>
+            <Flex.Item>
+              <span>
+                Electrical hazard authorised <br />
+                personnel only
```

Comment:
`Electrical Hazard • Authorised Personnel Only` imo

Reply:
This makes sense